lib/content-services/src/lib/breadcrumb/breadcrumb.component.html

```diff
@@ -55,7 +55,7 @@
             {{ item.name | translate }}
         </a>
 
-        <div *ngSwitchDefault class="adf-breadcrumb-item-current" role="heading" aria-level="2"
+        <div *ngSwitchDefault class="adf-breadcrumb-item-current"
```

Comment:
Why did you remove this role?

Reply:
role and aria-level added again, it was giving some error i think at that time.